src/components/Uploads/UploadStatusButton.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { CSSTransition } from 'react-transition-group';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowUp } from '@fortawesome/free-solid-svg-icons';
+import * as AppActions from 'components/App/AppActions';
+import * as AppConstants from 'components/App/AppConstants';
+import * as Selectors from './UploadsSelectors';
+
+const StyledStatusButton = styled.button`
+  position: absolute;
+  right: 5px;
+  bottom: 50px;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  background-color: #2c7be0;
+  border: none;
+  outline: none;
+
+  &.processing {
+    animation: pulse 3s infinite;
+  }
+
+  &.enter {
+    opacity: 0;
+  }
+
+  &.enter-active {
+    opacity: 1;
+    transition: opacity 200ms;
+  }
+
+  &.exit {
+    opacity: 1;
+  }
+
+  &.exit-active {
+    opacity: 0;
+    transition: opacity 200ms;
+  }
+
+  @keyframes pulse {
+    0% {
+      opacity: 1;
+    }
+    50% {
+      opacity: 0.5;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+`;
+
+/**
+ * @todo: we need to do this component more information
+ * it should represent uploading status state:
+ * - when success
+ * - when partial success
+ * - when has errors
```

Comment:
I don't understand what this `todo` is about.
Could you explain better what has been done and what to do?
Maybe create an issue on github instead of a `@todo`?
When should we do this todo? If we just leave it like it is, what is going to be "wrong" with this component?

Reply:
I would like to have an issue, instead of a `@todo` in the code, so I can prioritize it for before MVP or after MVP.
Nobody is going to touch this `@todo` because there is no task "do all the todos". Everybody is doing only issues from github.

Reply:
Make status button more informative:

https://github.com/DarthVanger/NeuralNotes/issues/90

Refresh note children when upload completed:

https://github.com/DarthVanger/NeuralNotes/issues/91